docs/buildah-add.md

```diff
@@ -52,8 +56,9 @@ buildah add containerID 'passwd' 'certs.d' /etc
 ### `.dockerignore`
 
 If the file .dockerignore exists in the context directory, `buildah copy` reads
```

Comment:
There's no context directory for `buildah add`...?

Reply:
I changed the text to indicate this only happens if you specify the --ignorefile option.  Currently there is no .dockerignore used in buildah add or buildah copy, if I understand correctly.  Adding the --ignorefile option will allow users to take advantage of this functionality.

Reply:
That's right, since they don't call Add() with a context directory.